src/ids/update_pci_usb

```diff
@@ -0,0 +1,28 @@
+#! /bin/bash
+
+# Script to update pci and usb id data.
+#
+# Just run it.
+
+[ -x convert_hd -a -d src ] || {
+  echo "this script must be run in the 'src/ids' directory"
+  exit 1
+}
+
+curl -o pci.ids http://pci-ids.ucw.cz/v2.2/pci.ids || {
```

Comment:
```suggestion
curl --fail -o pci.ids http://pci-ids.ucw.cz/v2.2/pci.ids || {
```

Reply:
Try:

```sh
curl -o pci.ids http://pci-ids.ucw.cz/v2.2/pci.ids-oops-wrong-filename 
```

Without `--fail`/`-f`, curl returns and pci.ids contains a 404 notice.